v3/trnthsn/s6xty/trnthsn_s6xty_via.json

```diff
@@ -0,0 +1,157 @@
+{
+	"name": "S6xty",
+	"vendorId": "0x5453",
+	"productId": "0x5336",
+	"menus": ["qmk_rgblight"],
+	"keycodes": ["qmk_lighting"],
+	"matrix": {
+		"rows": 5,
+		"cols": 14
+	},
+	"layouts": {
+		"labels": [
+			"Split Backspace",
+			["Right Shift", "Split", "Full", "Arrow"],
+			["Bottom Row", "Tsangan", "Arrow"]
+		],
+		"keymap": [
+			[
+				{
+					"y": 0.25
+				},
+				"0,0",
+				"0,1",
+				"0,2",
+				"0,3",
+				"0,4",
+				"0,5",
+				"0,6",
+				"0,7",
+				"0,8",
+				"0,9",
+				"0,10",
+				"0,11",
+				"0,12",
+				{
+					"w": 2
+				},
+				"0,13\n\n\n0,0",
+				{
+					"x": 0.25
+				},
+				"2,12\n\n\n0,1",
+				"0,13\n\n\n0,1"
+			],
+			[
+				{
+					"w": 1.5
+				},
+				"1,0",
+				"1,1",
+				"1,2",
+				"1,3",
+				"1,4",
+				"1,5",
+				"1,6",
+				"1,7",
+				"1,8",
+				"1,9",
+				"1,10",
+				"1,11",
+				"1,12",
+				{
+					"w": 1.5
+				},
+				"1,13"
+			],
+			[
+				{
+					"w": 1.75
+				},
+				"2,0",
+				"2,1",
+				"2,2",
+				"2,3",
+				"2,4",
+				"2,5",
+				"2,6",
+				"2,7",
+				"2,8",
+				"2,9",
+				"2,10",
+				"2,11",
+				{
+					"w": 2.25
+				},
+				"2,13"
+			],
+			[
+				{
+					"w": 2.25
+				},
+				"3,0",
+				"3,2",
+				"3,3",
+				"3,4",
+				"3,5",
+				"3,6",
+				"3,7",
+				"3,8",
+				"3,9",
+				"3,10",
+				"3,11\n\n\n1,0",
+				{
+					"w": 1.75
+				},
+				"3,12\n\n\n1,0",
+				"3,13\n\n\n1,0",
+				{
+					"x": 0.25
+				},
+				"3,11\n\n\n1,1",
+				{
+					"w": 2.75
+				},
+				"3,12\n\n\n1,1",
+				{
+					"x": 0.25,
+					"w": 1.75
+				},
+				"3,11\n\n\n1,2",
+				"3,12\n\n\n1,2",
+				"3,13\n\n\n1,2"
+			],
+			[
+				{
+					"w": 1.5
+				},
+				"4,0",
+				"4,1",
+				{
+					"w": 1.5
+				},
+				"4,2",
+				{
+					"w": 7
+				},
+				"4,7",
+				{
+					"w": 1.5
+				},
+				"4,10\n\n\n2,0",
+				"4,12\n\n\n2,0",
+				{
+					"w": 1.5
+				},
+				"4,13\n\n\n2,0",
+				{
+					"x": 4
+				},
+				"4,10\n\n\n2,1",
+				"4,11\n\n\n2,1",
+				"4,12\n\n\n2,1",
+				"4,13\n\n\n2,1"
+			]
+		]
+	}
+}
```

Comment:
```suggestion
{
  "name": "S6xty",
  "vendorId": "0x5453",
  "productId": "0x5336",
  "matrix": {
    "rows": 5,
    "cols": 14
  },
  "keycodes": [
    "qmk_lighting"
  ],
  "menus": [
    "qmk_rgblight"
  ],
  "layouts": {
    "labels": [
      "Split Backspace",
      [
        "Right Shift",
        "Split",
        "Full",
        "Arrow"
      ],
      [
        "Bottom Row",
        "Tsangan",
        "Arrow"
      ]
    ],
    "keymap": [
      [
        {
          "x": 13
        },
        "2,12\n\n\n0,1",
        {
          "c": "#aaaaaa"
        },
        "0,13\n\n\n0,1"
      ],
      [
        {
          "y": 0.25
        },
        "0,0",
        {
          "c": "#cccccc"
        },
        "0,1",
        "0,2",
        "0,3",
        "0,4",
        "0,5",
        "0,6",
        "0,7",
        "0,8",
        "0,9",
        "0,10",
        "0,11",
        "0,12",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "0,13\n\n\n0,0"
      ],
      [
        {
          "w": 1.5
        },
        "1,0",
        {
          "c": "#cccccc"
        },
        "1,1",
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        "1,8",
        "1,9",
        "1,10",
        "1,11",
        "1,12",
        {
          "w": 1.5
        },
        "1,13"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "2,0",
        {
          "c": "#cccccc"
        },
        "2,1",
        "2,2",
        "2,3",
        "2,4",
        "2,5",
        "2,6",
        "2,7",
        "2,8",
        "2,9",
        "2,10",
        "2,11",
        {
          "c": "#aaaaaa",
          "w": 2.25
        },
        "2,13"
      ],
      [
        {
          "w": 2.25
        },
        "3,0",
        {
          "c": "#cccccc"
        },
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        "3,7",
        "3,8",
        "3,9",
        "3,10",
        "3,11\n\n\n1,0",
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "3,12\n\n\n1,0",
        "3,13\n\n\n1,0",
        {
          "x": 0.25,
          "c": "#cccccc"
        },
        "3,11\n\n\n1,1",
        {
          "c": "#aaaaaa",
          "w": 2.75
        },
        "3,12\n\n\n1,1",
        {
          "x": 0.25,
          "w": 1.75
        },
        "3,11\n\n\n1,2",
        "3,12\n\n\n1,2",
        "3,13\n\n\n1,2"
      ],
      [
        {
          "w": 1.5
        },
        "4,0",
        "4,1",
        {
          "w": 1.5
        },
        "4,2",
        {
          "c": "#cccccc",
          "w": 7
        },
        "4,7",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "4,10\n\n\n2,0",
        "4,12\n\n\n2,0",
        {
          "w": 1.5
        },
        "4,13\n\n\n2,0"
      ],
      [
        {
          "y": 0.25,
          "x": 11
        },
        "4,10\n\n\n2,1",
        "4,11\n\n\n2,1",
        "4,12\n\n\n2,1",
        "4,13\n\n\n2,1"
      ]
    ]
  }
}

```

Reply:
Applied formatting. Pull the changes and modify the file name as suggested in the previous comment. Thx 👍